cmd/fleet/main.go

```diff
@@ -478,10 +481,10 @@ LOOP:
 		ech := make(chan error, 2)
 
 		if started {
-			f.reporter.Status(proto.StateObserved_CONFIGURING, "Re-configuring", nil)
+			_ = f.reporter.Status(proto.StateObserved_CONFIGURING, "Re-configuring", nil)
```

Comment:
I would prefer we disable the linter using comments (`nolint` instead of using the `_` to not handle the error.
Because we can search for `nolint` and do the right change here.

If we were handling the error here what would be the course of action?
- Fails?
- return error?
- Should we log it? and if yes at what log level? if it's not in debug is this situation actionable by the user?